pipelines/batch/src/main/java/com/google/fhir/analytics/FetchSearchPageFn.java

```diff
@@ -115,7 +115,9 @@ abstract class FetchSearchPageFn<T> extends DoFn<T, KV<String, Integer>> {
 
   private final int maxPoolSize;
 
-  private static final Cache<String, String> goldenResources = Caffeine.newBuilder().maximumSize(10000).build();;
+  private static final Cache<String, String> goldenResources =
+      Caffeine.newBuilder().maximumSize(10000).build();
+  ;
```

Comment:
2 `;;`?

Reply:
Are we good to remove fetch mode support?

Reply:
It does not scale, and we should remove the code we don't want to maintain.